pkg/accountmanager/iam.go

```diff
@@ -32,7 +32,7 @@ func (p *principalService) MergeRole() error {
 	})
 	if err != nil {
 		if isAWSAlreadyExistsError(err) {
-			log.Print(err.Error() + " (Ignoring)")
+			log.Print(err.Error() + fmt.Sprintf(" (Ignoring this error for account %q)", *p.account.ID))
```

Comment:
For readability, consider rewriting this to:

`log.Printf("%s: role already exists for account %q; ignoring", err.Error(), *p.account.ID)`

In other words, use the built-in `Printf` method.

Reply:
log.Printf("%s: for account %q; ignoring", err.Error(), *p.account.ID)